pkgs/development/python-modules/hebg/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # Runtime dependencies
+  networkx,
+  matplotlib,
+  numpy,
+  tqdm,
+  scipy,
+
+  # Build, dev and test dependencies
+  setuptools-scm,
+  pytestCheckHook,
+  pytest-check,
+  pytest-mock,
+}:
+
+buildPythonPackage rec {
+  pname = "hebg";
+  version = "0.2.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-11bz+FbnaEVLiXT1eujMw8lvABlzVOeROOsdVgsyfjQ=";
+  };
+
+  build-system = [ setuptools-scm ];
+
+  dependencies = [
+    networkx
+    matplotlib
+    numpy
+    tqdm
+    scipy
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-check
+    pytest-mock
+  ];
+
```

```suggestion
  env = lib.optionalAttrs stdenv.hostPlatform.isDarwin { MPLBACKEND = "Agg"; };
```
I guess this environment variable fixes tests on darwin.